run-integration-test/kustomize/bases/opentelemetry-collectors/container-log-scrape/opentelemetrycollector.yml

```diff
@@ -99,6 +99,8 @@ spec:
             on_error: send_quiet
             parse_from: body.message
 
+    extensions: {}
```

Comment:
question: Am I save to to assume this is only there so that it can be populated in the overlay? If yes, we should add a comment.

Reply:
Yes you are to assume that.

I will add comments to all of the explicitly empty KVs.

Reply:
c80df03